lib/resources/PhoneNumbersInboundChannels.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var TelnyxResource = require('../TelnyxResource');
+var utils = require('../utils');
+var telnyxMethod = TelnyxResource.method;
+
+function transformResponseData(response, telnyx) {
+  return utils.addResourceToResponseData(
+    response,
+    telnyx,
+    'phoneNumbersInboundChannels',
+    {
+      update: telnyxMethod({
+        method: 'PATCH',
+        path: '',
+        urlParams: ['channels'],
+        paramsValues: [response.data.id],
+        paramsNames: ['channels'],
+      }),
+    }
+  );
+}
+
+module.exports = require('../TelnyxResource').extend({
+  path: 'phone_numbers/inbound_channels',
```

Comment:
@romulogarofalo please also add this resource import to https://github.com/team-telnyx/telnyx-node/blob/master/lib/resources/PhoneNumbers.js#L12